p4runtime_sh/shell.py

```diff
@@ -340,23 +340,27 @@ def _ipython_key_completions_(self):
     def __dir__(self):
         return ["clear"]
 
-    def _get_mf(self, name):
+    def _full_field_name(self, name):
         if name in self._fields:
-            return self._fields[name]
+            return name
         if name in self._fields_suffixes:
-            return self._fields[self._fields_suffixes[name]]
+            return self._fields_suffixes[name]
         raise UserError(
             "'{}' is not a valid match field name, nor a valid unique suffix, "
             "for table '{}'".format(name, self._table_name))
 
+    def _get_mf(self, name):
+        fullname = self._full_field_name(name)
+        return self._fields[fullname]
+
     def __setitem__(self, name, value):
         field_info = self._get_mf(name)
         self._mk[name] = self._parse_mf(value, field_info)
         print(self._mk[name])
 
     def __getitem__(self, name):
-        _ = self._get_mf(name)
-        print(self._mk.get(name, "Unset"))
+        fullname = self._full_field_name(name)
+        return self._mk.get(fullname, "Unset")
```

Comment:
while I agree that returning a value makes more sense, I feel like `"Unset"` makes more sense as something you display to the user in the context of the shell, and not as a return value (I feel like `None` would be better there), so I suggest something like this:
```python
    def __getitem__(self, name):
        fullname = self._full_field_name(name)
        f = self._mk.get(fullname, None)
        if f is None:
            print("Unset")
        return f
```

Reply:
Agreed, and done.